feat: remove-createPolkadotJsScClient (#909)
paritytech/substrate-connect: packages/connect/CHANGELOG.md

## [Unreleased]

### Breaking

> - Removed `createPolkadotJsScClient` from the API, because the `ScProvider` has been added to `@polkadot/rpc-provider`

Reviewer: ```suggestion
- Removed `createPolkadotJsScClient` from the API, as the `ScProvider` has been upstreamed to `@polkadot/rpc-provider`
```

Author: Thanks!
